src/routes/+page.svelte

```diff
@@ -188,12 +188,16 @@
         </p>
       </header>
       <div class="flex flex-row flex-wrap items-start justify-around">
-        <ProjectCell href="https://Pome.gr/">
+        <ProjectCell href="https://overlord.fund/">
+          <span slot="title">Overlord</span>
+          <span slot="description">One-stop-shop for investment funds back-office</span>
```

Comment:
Dit comme ça j'ai l'impression que ça veut dire "Solution tout en un pour les Backoffices de fonds d'investissements".

J'ai l'impression que comme ça ce serait plus clair :

```suggestion
          <span slot="description">The one-stop-shop Back-office for investment funds</span>
```

Je sais pas ce que t'en penses ?

Reply:
Sounds good!